src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/parser/ActionMetricsParserTest.kt

```diff
@@ -77,4 +79,51 @@ class ActionMetricsParserTest {
             equalTo(ofSeconds(1) + ofMillis(637))
         )
     }
-}
\ No newline at end of file
+
+    @Test
+    fun shouldParseCompact() {
+        val metricsStream = javaClass.classLoader.getResourceAsStream("virtual-user-results/1/action-metrics.jpt")
+
+        val parser = ActionMetricsParser(MetricCompactJsonFormat())
+
+        @Suppress("Deprecation")
```

Comment:
Is not deprecated API available?

Reply:
Yeah, we can always `parser.stream(it).toList()`